### DISCUSSION TITLE
Saleor as a ticketing system

### DISCUSSION THREAD

**Discussion by Rafcin** [saleor/saleor#9454] [Q&A]
Hi, I've been looking at Saleor for a while as a solution for a project, and I wanted to open this discussion to ask how well could Saleor be used as an event ticketing system. I want to use Saleor to allow people to buy tickets for a set of events before the event and at the event. I think Saleor would be a good solution, however, I have a few questions.

The first question is, could I use Saleor offline? One of the biggest issues for my use case would be that sometimes at some of these events, the internet either isn't there or is so overloaded due to the number of people it's impossible to connect to any service. As a counter to this, my idea was to deploy Saleor to AWS and then have an onsite device with Saleor deployed as well and use Postgres BDR to handle any conflicts. I'm not sure if it's the greatest idea but I'm curious to hear if it would be possible and as well if anyone here has dealt with having to use Saleor online and offline. 

My second question is, can I remove all the shipping aspects from the system, since this would be a ticketing system I would see no need to keep the shipping backend and I'm curious if removing it would be painless or would it interfere with the entire design?

**Comment by karolkielecki** [comment]
Hi @Rafcin, 

Let me start with some good news. Re removing shipping aspects--you can create digital products. An example of a digital product is here: https://demo.saleor.io/default-channel/en-US/products/lake-tunes

Re using Saleor offline -- Saleor doesn't support such a use case out of the box. Tbh, I haven't heard about such a use case. I'm not sure what I can recommend to you here. Maybe you can create an app that tries to connect multiple times with Saleor until it's connected. Having separate db onsite will cause you many issues, including synchronization issues. Maybe you can create POS?
